From: Alexis Murzeau <amubtdx@gmail.com>
Date: Thu, 31 Dec 2020 13:02:06 +0100
Subject: Revert "docs: use recommonmark as an extension"

Debian Buster has recommonmark 0.4.0 so we need to revert this
modification.

Forwarded: not-needed
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs/conf.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 6b565b6..979bd82 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -23,7 +23,6 @@ extensions = [
     'ext_argparse',
     'ext_github',
     'ext_releaseref',
-    'recommonmark',
     'sphinx_rtd_theme'
 ]
 
@@ -32,6 +31,12 @@ autosectionlabel_prefix_document = True
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
+# The suffix of source filenames.
+source_suffix = ['.rst', '.md']
+source_parsers = {
+   '.md': 'recommonmark.parser.CommonMarkParser',
+}
+
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
